src/main/java/com/gluonhq/substrate/target/MacOSTargetConfiguration.java

```diff
@@ -143,13 +143,30 @@ String getAdditionalSourceFileLocation() {
 
     @Override
     public boolean packageApp() throws IOException, InterruptedException {
-        createAppBundle();
+        boolean sign = !projectConfiguration.getReleaseConfiguration().isSkipSigning();
+        createAppBundle(sign);
+
+        String packageTypes = projectConfiguration.getReleaseConfiguration().getPackageTypes();
+        if (packageTypes == null || packageTypes.isEmpty()) {
+            // if it is not set, do nothing
+            return true;
+        } else if (!(packageTypes.contains("pkg") || packageTypes.contains("dmg"))) {
```

Comment:
shouldn't this be seen as a configuration error, hence throw an error?

Reply:
Since `packageTypes` can contain other types (for instance for Win or Linux), without the need of a profile, I was thinking that this shouldn't fail with an error.
But (and related to your other comment), it might be better to refactor this and force a single package type per run, and enforce the use of profiles. Anyway even `pkg` and `dmg` might need different properties that won't fit as they are now (version code, ...).